run_in_kubernetes.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 kubectl run sampleloader -it --rm \
     --generator=run-pod/v1 \
-    --image eu.gcr.io/census-rm-ci/census-rm-sample-loader \
+    --image eu.gcr.io/census-rm-ci/rm/census-rm-sample-loader \
```

Comment:
Maybe needs to be configurable since some environments should only be using census-gcr-rm images (but likewise some environments can't)